frontend/test/metabase/visualizations/components/settings/ChartSettingOrderedSimple.unit.spec.js

```diff
@@ -27,6 +27,6 @@ describe("chartsettingorderedsimple", () => {
 
     expect(screen.getByText("foo")).toBeVisible();
     expect(screen.getByText("bar")).toBeVisible();
-    expect(screen.queryByText("another")).not.toBeVisible();
+    expect(screen.queryByText("another")).not.toBeInTheDocument();
```

Comment:
Again, since we're not rendering the element anymore, I needed to change the assertion